lib/ansible/modules/cloud/digital_ocean/digital_ocean_sshkey_info.py

```diff
@@ -79,9 +81,9 @@ def core(module):
     status_code = response.status_code
     json = response.json
     if status_code == 200:
-        module.exit_json(changed=False, ansible_facts=json)
+        module.exit_json(changed=False, data=json)
```

Comment:
You can't just change the return value, this will break people's playbooks. You need to return `ansible_facts` when the module is called with the old name, and the new stuff when it is called with the new name.

If you want to rename the result, it's best to simply deprecate the old module and create a new module (which will also have new documentation, and where the new return value(s) can be properly documented without removing docs for the old ones.)

Reply:
True, was trying to make it consistent with the other modules and missed that :/

Reply:
I will keep the value as it was. All modules are returning **data** apart from _floating_ips_ and _sshkeys_ tho.
Does anyone has different opinion on this?

Reply:
The problem is that you cannot use `ansible_facts` for the `_info` module. Everything returned in `ansible_facts` will be made available globally.

So maybe it is better to deprecate the `_facts` module and replace it with a new `_info` module which uses `data` (as the other modules)?

Reply:
@felixfontein should I do what was done for instance with AWS modules https://github.com/ansible/ansible/pull/60178/files ?
They updated the new documentation (I assume a release doesn't change old doc anyway) and with a simple condition the module returns a different name.

Reply:
Have both versions ready to push, just need to decide which one :/

Reply:
I would proceed as in #60178 for `lambda_facts`: I deprecated `lambda_facts`, and created a new module `lambda_info`, which is essentially a copy of `lambda_facts`, except that the name of the return value changed.

Reply:
> They updated the new documentation (I assume a release doesn't change old doc anyway) and with a simple condition the module returns a different name.

The modules in #60178 (except `lambda_facts`/`lambda_info`) return the result by the same name (everything inside `ansible_facts` is the actual name(s) of the result). That's why I was able to simply rename them, and let them return the contents of `ansible_facts` directly when called as `_info`. But for `lambda_facts`, the name used inside `ansible_facts` wasn't suitable, so I had to deprecate the module and create a new one.

Reply:
Hope I didn't missed anything :/

Reply:
I meant this one for instance, we could do the samething here https://github.com/ansible/ansible/pull/60178/files#diff-076fbee0e72cc8b32df04fb66e17cf0eR112

Reply:
Anyway, should be solved now :)

Reply:
The method from the other PR won't work because of documentation. But with the current state of the PR, it should be fine :)